event_spec.md

```diff
@@ -270,17 +289,154 @@ nodeName  | Name of the Worker node or Compute node where the producer exists.
 resource | The hierarchical name for the resource.  For this specification, the resource hierarchy will align with the yang models specified in O-RAN.WG4.MP-Yangs-v05.00 to the extent possible.  A * may be specified in the address hierarchy to subscribe to events below the specified level.| ../o-ran-sync/sync-group/sync-status/sync-state , ../o-ran-sync/sync-group/*| 
 
 
-### value_type
+#### value_type
 - `value_type`
     The type format of the [`value`](#value) property.
 
 
-### value
+#### value
 - `value`
   String representation of value in value_type format
 
+### Data
+Data represents a Redfish Event as defined in Redfish schema
```

Comment:
here Data represent either PTP event schema or HW right ?

Reply:
`Data` here is `Data.Data` which is HW only.

Reply:
You should call it value since RedfishEvent type is a value of data type .

Reply:
I plan to change this in the future. See proposed change for issue https://github.com/redhat-cne/sdk-go/issues/24